getting_started/step_by_step/signals.rst

```diff
@@ -203,6 +203,50 @@ To emit a signal via code, use the ``emit_signal`` function:
             EmitSignal(nameof(MySignal));
         }
     }
+    
+A signal can also pass on a value, add it in parent:
```

Comment:
Did you mean "parentheses" here? Note that a signal can pass any number of arguments, not just "a" value. Also, declaring them is optional. How about using the same wording as in the GDScript doc:

> You can write optional argument names in parentheses in the signal’s definition.